pkg/engine/engine.go

```diff
@@ -115,6 +115,8 @@ func (e *Engine) Render(chrt *chart.Chart, values chartutil.Values) (map[string]
 type renderable struct {
 	// tpl is the current template.
 	tpl string
+	// path is the path of chart
+	path string
```

Comment:
`basePath` is a physical path of a template file like `templates/xxx.yaml` or 'charts/chartX/templates/xxx.yaml`.
`path` is a logic path of a chart like `chartA/chartB/chartC`. The `path` stands for a hierarchy of subcharts.